slangpy/tests/device/test_buffer_cursor.py

```
@@ -395,6 +395,59 @@ def test_cursor_read_write(device_type: spy.DeviceType, seed: int):
+    )
+    reinterpret_type_layout = module.layout.get_type_layout(
+        module.layout.find_type_by_name("uint16_t[2]")
+    )
+
+    # Create a buffer cursor with its own data
+    cursor = spy.BufferCursor(device_type, resource_type_layout.element_type_layout, 1)
+    
+    # Populate the first element
+    element = cursor[0]
+    for test in tests:
+        (name, gpu_type, gpu_val, value) = test[0:4]
```

Ruff (0.15.12)

[warning] Unpacked variable `gpu_type` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)